scripts/notify-subscribers-of-breach.js

```diff
@@ -5,7 +5,7 @@ if (!breachName) {
   throw("Usage: node notify-subscribers-of-breach.js \"breach-name\"");
 }
 
-const DBUtils = require("../db/utils");
+const DBUtils = require("../db/db");
```

Comment:
Should we bother renaming this to `DB` as well, to be consistent with the other files.
I can't imagine caring much either way, it's just that now I've seen it, so I feel legally obliged to say it.

Reply:
+1, let's call this DB now.